assets/app/scripts/services/logLinks.js

```diff
@@ -50,10 +50,37 @@ angular.module('openshiftConsole')
         newTab(params);
       };
 
-      // Not currently used.
-      // new tab: path/to/current?view=textonly
-      var textOnlyLink = function() {
-        newTab({view: 'textonly'});
+      // broken up for readability:
+      var template = new URITemplate([
+        "/#/discover?",
+        "_g=(",
+          "time:(",
+            "from:now-1w,",
+            "mode:relative,",
+            "to:now",
+          ")",
+        ")",
+        "&_a=(",
+          //"columns:!(_source),",
+          "columns:!(kubernetes_container_name,{containername}),",
+          "index:'{namespace}.*',",
+          "query:(",
+            "query_string:(",
+              "analyze_wildcard:!t,",
+              "query:'kubernetes_pod_name: {podname} %26%26 kubernetes_namespace_name: {namespace}'",
+            ")",
+          "),",
+          "sort:!(time,desc)",
+        ")",
+        // NOTE: slightly older versions of kibana require openshift_ prefix, not console_
```

Comment:
We can probably remove this note since no one should be running those older versions.